x-pack/plugins/actions/server/lib/create_action_event_log_record_object.ts

```diff
@@ -68,6 +78,10 @@ export function createActionEventLogRecordObject(params: CreateActionEventLogRec
       ...(task ? { task: { scheduled: task.scheduled, schedule_delay: task.scheduleDelay } } : {}),
     },
     ...(message ? { message } : {}),
+    action: {
+      name,
+      uuid: uuid(),
```

Comment:
why are we adding a uuid here?

Reply:
I see why we are adding a UUID but I think the UUID should be generated in the action executor and passed to this helper function instead of generated here. That way all event log docs written during an action execution will have the same UUID.

Reply:
Resolved in this commit efc5a506870cb13ad58e467d52ea5f6bf2812af9